spec/spec_helper.rb

```diff
@@ -69,13 +69,13 @@
   # end of the spec run, to help surface which specs are running
   # particularly slow.
 =end
-  config.profile_examples = 10
+  # config.profile_examples = 10
 
   # Run specs in random order to surface order dependencies. If you find an
   # order dependency and want to debug it, you can fix the order by providing
   # the seed, which is printed after each run.
   #     --seed 1234
-  config.order = :random
+  # config.order = :random
```

Comment:
No, it just disturbed me when I ran single spec files (I wanted to see the order).